daq/host.py

```diff
@@ -352,7 +357,7 @@ def _base_tests(self):
             self.record_result('base', exception=e)
             self._monitor_cleanup()
             raise
-        self.record_result('base')
+        self.record_result('base', state='pass')
```

Comment:
I dont think I understand where we use the new state information being passed

Reply:
Ah yes. It's routed through to the cloud (record_result eventually calls push_message or something similar). Roughly described here https://github.com/faucetsdn/daq/blob/master/docs/firebase.md.